notes/disaster-food-replacement.vbs

```diff
@@ -44,106 +44,200 @@ changelog = array()
+    IF trim(worker_signature) = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
+  LOOP UNTIL err_msg = "" and ButtonPressed <> TSS_BENE_webform_btn									'loops until all errors are resolved
+  CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
 Loop until are_we_passworded_out = false					'loops until user passwords back in
 
+'Create TIKL if selected
+If dhs_1609_tikl = CHECKED Then Call create_TIKL("DHS-1609 was sent 10 days ago and is now due back.", 10, date, False, TIKL_note_text)
 
+'Write CASE/NOTE with information
 start_a_blank_case_note
-'writes case note for Baby Born
-CALL write_variable_in_Case_Note("--Food Destroyed in a Disaster Reported - " & replacement_status & "--")
-CALL write_bullet_and_variable_in_Case_Note("Type of Disaster", disaster_type)
-CALL write_bullet_and_variable_in_Case_Note("How the disaster was verified", how_verif)
-CALL write_bullet_and_variable_in_Case_Note("Date client reported the loss of food to county", report_date)
+CALL write_variable_in_Case_Note("--Food Destroyed in Disaster Reported - " & replacement_status & "--")
 CALL write_bullet_and_variable_in_Case_Note("Date of Loss", loss_date)
 CALL write_bullet_and_variable_in_Case_Note("Amount of Food Loss", amount_loss)
+CALL write_bullet_and_variable_in_Case_Note("Date client reported the loss of food to county", report_date)
+CALL write_bullet_and_variable_in_Case_Note("Description of Disaster", disaster_description)
+CALL write_bullet_and_variable_in_Case_Note("How the disaster was verified", how_verif)
+CALL write_bullet_and_variable_in_Case_Note("Date Loss Verified", loss_verification_date)
 CALL write_bullet_and_variable_in_Case_Note("Replace as REI", rei_replacement)
-IF TSS_BENE_sent_checkbox <> CHECKED THEN CALL write_bullet_and_variable_in_Case_Note("Stauts of Request", replacement_status)
-CALL write_bullet_and_variable_in_Case_Note("Reason for Denial", denial_reason)
-CALL write_bullet_and_variable_in_Case_Note("Verifcations Requested", verif_needed)
+CALL write_bullet_and_variable_in_Case_Note("Status of Request", replacement_status)
+IF replacement_status = "Request Denied" Then CALL write_bullet_and_variable_in_Case_Note("Reason for Denial", denial_reason)
+CALL write_bullet_and_variable_in_Case_Note("Verifications Requested", verif_needed)
 IF TSS_BENE_sent_checkbox = CHECKED THEN CALL write_variable_in_Case_Note("* Submitted a TSS BENE request (webform) through SIR")
 CALL write_variable_in_Case_Note("Nonreceipt/Replacement Affidavit (DHS-1609)")
+CALL write_variable_in_Case_Note(" Date DHS-1609 is due back from the client: " & dhs_1609_due_date)
+If dhs_1609_tikl = CHECKED Then write_variable_in_Case_Note(" TIKL created for DHS-1609 due date.")
 CALL write_variable_in_Case_Note(" Date DHS-1609 was signed by the client: " & dhs1609_sig_date)
```

Comment:
Could some of these be updated to only note if there is content?
If dhs1609_sig_date <> "" Then Call write_variable...

Reply:
Updated to only write non-blank variables